components/Layout/index.js

```diff
@@ -1,10 +1,13 @@
 import styled from 'styled-components';
+import Navigation from '../NavigationBar';
 
 export default function Layout() {
   return (
+    <>
     <StyledHeader>
       <h1>My Solar Calculator</h1>
     </StyledHeader>
+    </>
```

Comment:
You don't need fragments here:

```suggestion
    <StyledHeader>
      <h1>My Solar Calculator</h1>
    </StyledHeader>
```